trl/trainer/prm_config.py

```diff
@@ -46,6 +46,11 @@ class PRMConfig(TrainingArguments):
             Whether to train only on the last step.
         dataset_num_proc (`int`, *optional*, defaults to `None`):
             Number of processes to use for processing the dataset.
+        average_tokens_across_devices (`bool`, *optional*, defaults to `True`):
+            Whether or not to average tokens across devices. If enabled, will use all_reduce to synchronize
+            num_tokens_in_batch for precise loss calculation. Reference:
+            https://github.com/huggingface/transformers/issues/34242
+            Overrides the `average_tokens_across_devices` parameter in [`~transformers.TrainingArguments`].
```

Comment:
Starting from https://github.com/huggingface/trl/pull/3530, we stop overriding the docstring from `TrainingArgument`. It make things more concise, and easy to read. We just mention

> Note that default values in this class may differ from those in [`~transformers.TrainingArguments`].

The new default are displayed anyway in the TRL doc.

```suggestion
```

Reply:
<img width="1043" alt="Screenshot 2025-06-04 at 11 57 42" src="https://github.com/user-attachments/assets/0a66aa5c-1c70-4ba4-9fc6-863b338c7d8d" />